ReleaseNotes.md

```diff
@@ -1,5 +1,56 @@
 # Release Notes
 
+## New in Release 2.5.0
```

Comment:
Not covered: 
- PTQ support for PyTorch: added MinMax and FBC algorithms
- prepare_for_inference API 
- JPQD support for BERT, Wave2VecV2, Swin, ViT, DistilBERT, CLIP, MobileBERT

Reply:
resolved

Reply:
> * PTQ support for PyTorch: added MinMax and FBC algorithms

Added, only MinMax algo
> * prepare_for_inference API

It's added as `strip()`.

> * JPQD..

Added.